cmd/minishift/cmd/start_preflight.go

```diff
@@ -107,7 +107,7 @@ func preflightChecksBeforeStartingHost() {
 		checkIsoUrl,
 		"Checking the ISO URL",
 		false, configCmd.WarnCheckIsoUrl.Name,
-		"See the 'Managing Minishift' topic for more information")
+		"See the 'Managing Minishift' topic (https://docs.openshift.org/latest/minishift/using/managing-minishift.html) for more information")
```

Comment:
The topic ` managing-minishift` has changed to https://github.com/minishift/minishift/blob/master/docs/source/using/basic-usage.adoc. Once we release a new version of Minishift (which is today) the docs at https://docs.openshift.org/latest/minishift/index.html